codegen-winter/src/air/boundary_constraints.rs

```diff
@@ -98,14 +98,14 @@ impl Codegen for BoundaryExpr {
         match self {
             Self::Const(value) => {
                 if is_aux_constraint {
-                    format!("E::from({})", value)
+                    format!("E::from({} as u64)", value)
```

Comment:
This cast shouldn't be necessary, since we know that value is always a u64. We should use "_u64" instead of casting so that it is defined with the correct type in the first place, e.g.:

```
format!("E::from({}_u64)", value)
```

This should be updated in all equivalent cases where the `as u64` cast is used throughout these air changes

Reply:
Fixed, thanks.